circle.yml

```diff
@@ -1,6 +1,6 @@
 machine:
   node:
-    # version: 0.11.8
+    version: 7.2
```

Comment:
I'm more than open to put a higher version, like the latest 7.8. Is there a reason we want to stay on a non-latest version ?